vendor/assets/stylesheets/ustyle/components/_lists.scss

```diff
@@ -39,8 +44,15 @@
     margin-top: 1em;
 
     a {
-      font-size: em(18px);
       color: $c-navy;
     }
   }
 }
+
+@include respond-to(mobile) {
```

Comment:
Is this not the same as above? Also for `ustyle` can we maintain the consistency of `respond-to` and media queries within the selectors rather than at the bottom of the Sass files. If we want to rewrite we'll have to revisit as a whole.

Reply:
This was being overridden elsewhere, which no longer seems to be the case so I have remove it.